src/main/java/io/jenkins/plugins/conventionalcommits/ConventionalCommits.java

```diff
@@ -25,4 +25,15 @@ public Version nextVersion(Version in, List<String> commits) {
 
         return in.incrementPatchVersion();
     }
+
+    private boolean breakingChangeFooter(String commit){
+        int startIndex = commit.lastIndexOf("\n");
+
+        String result = commit;
+
+        if(startIndex!=-1 && startIndex!= commit.length()){
+            result = commit.substring(startIndex+1);
+        }
+        return result.startsWith("BREAKING CHANGE");
+    }
```

Comment:
Hey @jonyto123, thank you for your contribution.

I am afraid this logic fails for commits having multiline/multiple footers or just more than one "\n"s
For example for a commit message like:

```
feat: some feature

This commit adds a new feature but its a breaking change.

BREAKING CHANGE: Going from nothing to something, meaning anyone doing
nothing before suddenly has something to do. That sounds like a change
in your break.

Co-Authored-By: abc <abc@xyz.com>
Closes #someIssueNumber
```

Reply:
Should i decline the pr or you assume it from now? I do not have a clear way to make correct implementation.